src/ioctls/system.rs

```diff
@@ -22,6 +22,8 @@ use vmm_sys_util::errno;
 use vmm_sys_util::ioctl::ioctl_with_mut_ptr;
 use vmm_sys_util::ioctl::{ioctl, ioctl_with_val};
 
+const KVM_MAX_CPUID_ENTRIES: usize = 80;
```

Comment:
We don't need to redefine this here. It is already defined in `kvm-bindings`. To use it, we can import it: `use kvm_bindings::KVM_MAX_CPUID_ENTRIES`.